cmd/rect_test/main.go

```diff
@@ -0,0 +1,79 @@
+package main
```

Comment:
The "Tests" checkbox is actually for a test that would pass via `go test`, not a binary to manually check the result. Actually what is in this file should likely be included in fyne_demo/canvas section.

Reply:
Indeed. This would have to be removed as we shouldn't add more applications into the cmd/ directory.

Reply:
Yes, this is still an open point. I haven't had time yet.
Next steps will be:
- Test implementation
- Extend fyne_demo
- Delete cmd/rect_test

In the meantime you can use the rect_test for web and mobile testing.

Reply:
Open points done.